backend/curriculum_tracking/tests/test_RecruitProject.py

```diff
@@ -82,3 +86,75 @@ def test_too_long(self):
         self.assertIn("-test-project-", repo_name)
         self.assertIn(self.user.first_name, repo_name)
         self.assertIn(self.user.last_name, repo_name)
+
+
+class get_duration_Tests(TestCase):
+
+    def make_project_card(self, status):
+        self.card = factories.AgileCardFactory(
```

Comment:
Why is this getting complicated again? 

Why are cards here at all? 

Write loosely coupled code. Dont write code that relied on things it doesn't need. This stuff will break.